pkgs/by-name/wa/watchdogd/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, autoreconfHook
+, libite
+, libuev
+, libconfuse
+}:
+stdenv.mkDerivation rec {
+  pname = "watchdogd";
+  version = "3.5";
+
+  src = fetchFromGitHub {
+    owner = "troglobit";
+    repo = "watchdogd";
+    rev = version;
```

nit: consider using the finalAttrs pattern. It's supposed to be better in some ways, but I'm not sure about the specifics.

```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "watchdogd";
  version = "3.5";

  src = fetchFromGitHub {
    owner = "troglobit";
    repo = "watchdogd";
    rev = finalAttrs.version;
```

---

finalAttrs helps while overriding as it'll update when you override the version, but I think it's not worth it here, since the only relevant place is in the fetcher, where the hash will change.

---

Sounds fine by me. It's trivial to fix it up, if it ever becomes an issue again or if someone makes a treewide pr.